packages/eas-cli/src/commands/submit.ts

```diff
@@ -107,6 +107,11 @@ export default class BuildSubmit extends EasCommand {
       options: ['completed', 'draft', 'halted', 'inProgress'],
       helpLabel: ANDROID_FLAGS,
     }),
+    'changes-not-send-for-review': flags.boolean({
+      description:
+        '[default: false] Indicates that the changes in this edit will not be reviewed until they are explicitly sent for review from the Google Play Console UI',
```

Comment:
```suggestion
        '[default: false] Indicates that the changes sent with this submission will not be reviewed until they are explicitly sent for review from the Google Play Console UI',
```
? 🤔  I have no idea how to better phrase this.

_FYI for reviewers who don't know where this message came from: [Google API docs](https://developers.google.com/android-publisher/api-ref/rest/v3/edits/commit#query-parameters)_